Blockzilla/BrowserViewController.swift

```diff
@@ -747,6 +785,15 @@ extension BrowserViewController: WebControllerDelegate {
     func webController(_ controller: WebController, stateDidChange state: BrowserState) {}
 
     func webController(_ controller: WebController, didUpdateTrackingProtectionStatus trackingStatus: TrackingProtectionStatus) {
+        // Calculate the number of trackers blocked and add that to lifetime total
+        if case .on(let info) = trackingStatus {
+            if case .on(let oldInfo) = trackingProtectionStatus {
+                let numberOfAdditionalTrackersBlocked = max(0, info.total - oldInfo.total)
```

Comment:
This took me a minute to understand what/why we're doing it this way. I wonder if we named it something like `differenceSinceLastUpdate` it would be more clear?